.gitattributes

```diff
@@ -1,8 +1,6 @@
-# Declare SVG files to have LF endings on checkout
-# On Windows, the default git setting for `core.autocrlf`
-# means that when checking out code, LF endings get converted
-# to CRLF.
-# *.svg text eol=lf
```

Comment:
This setting has been commented out, so I removed it.

Reply:
I think this was meant to be left in as an illustration for Windows users but `text=auto` will hopefully take care of this too.